zerver/migrations/0344_create_realm_user_default_table.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.2 on 2021-06-01 16:19
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def create_realm_user_default_table(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    RealmUserDefault = apps.get_model("zerver", "RealmUserDefault")
+    # We exclude the "SYSTEM_BOT_REALM" here because this realm only contains
+    # Zulip "system bots" and will have no use of this table.
```

Comment:
This comment should be deleted, since it's no longer accurate.

Reply:
Done